index.html

```diff
@@ -21,14 +21,15 @@
   <header>
     <div class="container header-background-color">
       <div class="row">
-        <div class="column-full column-half">
+        <div class="column-full column-half nav-height">
           <h3>Code Journal</h3>
+          <button class="navbar-button">Entries</button>
         </div>
       </div>
     </div>
   </header>
   <main>
-    <div data-view="entry-form">
+    <div class="data-view" data-view="entry-form">
```

Comment:
This class name will get very confusing alongside your `data-view` attribute. I would rename this to `view`